### DISCUSSION TITLE
Need to add check box in the cell !

### DISCUSSION THREAD

**Discussion by bushan1891** [exceljs/exceljs#2533] [Ideas & Proposals]
Hey, Guys I need to add a checkbox in the row .

is there a feature in this build which lets us have the checkbox  ?

any feedback on this is appreciated

**Comment by inmenou** [comment]
need this now, too

**Comment by calhaudev** [comment]
Hi guys, there is any update on this?

**Comment by yogithesymbian** [comment]
all you guys have trick or alternative for this ?

**Comment by seanhurwitz** [comment]
Anyone? I suppose a dropdown data validation with True/False or something like that would work, just doesn't have checkbox look and feel

**Comment by PedroCost4** [comment]
any updated on this? really need it

**Comment by Andre2553** [comment]
Any updates on this?
